livedev/livedev

```diff
@@ -131,6 +136,15 @@ class Remote(object):
             for action in actions:
                 action.run(self)
 
+    def create_ssh_id_if_not_exists(self):
+        cmd = 'ssh-keygen -t rsa -b 2048 -P "" -f ~/.ssh/id_rsa <<< n'
+        subprocess.run(cmd, shell=True)
+
+    def copy_ssh_id(self):
+        self.create_ssh_id_if_not_exists()
+        cmd = 'ssh-copy-id -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -i ~/.ssh/id_rsa.pub -f '+self.host
```

Comment:
Please use the array format for subprocess as it's done above.
Also do not use `shell=` because it's unsafe and definetly not needed here.
Last note is to use f-strings instead of string cocatenation.